suite-native/navigation/src/components/ScreenHeader.tsx

```diff
@@ -1,92 +1,47 @@
-    'activeStep' | 'numberOfSteps'
->;
+type ScreenHeaderProps = {
+    hasBottomPadding?: boolean;
+};
 
-const ICON_SIZE = 48;
+const SCREEN_HEADER_HEIGHT = 56;
 
-const screenHeaderStyle = prepareNativeStyle(utils => ({
+const switchWrapperStyle = prepareNativeStyle(utils => ({
     flexDirection: 'row',
     alignItems: 'center',
-    justifyContent: 'space-between',
-    height: ICON_SIZE,
-    marginHorizontal: utils.spacings.medium,
-}));
-
-const iconWrapperStyle = prepareNativeStyle(() => ({
-    width: ICON_SIZE,
-    height: ICON_SIZE,
-}));
-
-const headerContentStyle = prepareNativeStyle(_ => ({
-    height: ICON_SIZE,
-    alignItems: 'center',
-    justifyContent: 'center',
+    flex: 1,
+    height: SCREEN_HEADER_HEIGHT,
+    paddingVertical: utils.spacings.small,
+    paddingHorizontal: utils.spacings.medium,
+    borderColor: utils.colors.borderOnElevation1,
+    borderWidth: utils.borders.widths.large,
+    borderRadius: utils.borders.radii.round,
+    backgroundColor: utils.colors.backgroundSurfaceElevation1,
 }));
 
-const GoBackIcon = () => {
-    const navigation = useNavigation();
-    return (
-        <IconButton
-            iconName="chevronLeft"
-            size="medium"
-            colorScheme="tertiaryElevation0"
-            onPress={() => navigation.goBack()}
-        />
-    );
-};
-
-export const ScreenHeader = ({
-    leftIcon,
-    rightIcon,
-    style,
-    content,
-    numberOfSteps,
-    activeStep,
-    titleVariant = 'titleSmall',
-    hasGoBackIcon = true,
-}: ScreenHeaderWithIconsProps) => {
+export const ScreenHeader = ({ hasBottomPadding }: ScreenHeaderProps) => {
     const { applyStyle } = useNativeStyles();
 
-    const LeftIcon = hasGoBackIcon && !leftIcon ? <GoBackIcon /> : leftIcon;
-    const isStepperDisplayed = activeStep && numberOfSteps;
-
     return (
-        <View style={[applyStyle(screenHeaderStyle), style]}>
-            <View style={applyStyle(iconWrapperStyle)}>{LeftIcon}</View>
-            <Box flex={1} marginHorizontal="large" alignItems="center">
-                <VStack style={applyStyle(headerContentStyle)} spacing="small">
-                    {isStepperDisplayed && (
-                        <StepsProgressBar numberOfSteps={numberOfSteps} activeStep={activeStep} />
-                    )}
-                    {typeof content === 'string' ? (
-                        <Text variant={titleVariant} adjustsFontSizeToFit numberOfLines={1}>
-                            {content}
-                        </Text>
-                    ) : (
-                        content
-                    )}
-                </VStack>
-            </Box>
-
-            <View style={applyStyle(iconWrapperStyle)}>{rightIcon}</View>
-        </View>
+        <ScreenHeaderWrapper marginBottom={hasBottomPadding ? 'small' : undefined}>
+            <HStack alignItems="center">
+                <Box
+                    justifyContent="space-between"
+                    alignItems="center"
```

Comment:
why not to have those two in style as well?

Reply:
[d9d9f59](https://github.com/trezor/trezor-suite/pull/9629/commits/d9d9f59209908a8f01723675dcb0fedcdb988b53)